src/main/scala/org/broadinstitute/dsde/agora/server/business/AgoraBusiness.scala

```diff
@@ -10,7 +10,7 @@ import org.broadinstitute.dsde.agora.server.model.AgoraEntity
 object AgoraBusiness {
 
   def agoraUrl(entity: AgoraEntity): String = {
-    AgoraConfig.methodsUrl + entity.namespace.get + "/" + entity.name.get + "/" + entity.snapshotId.get
+    AgoraConfig.methodsUrl + entity.namespace.getOrElse("") + "/" + entity.name.getOrElse("") + "/" + entity.snapshotId.getOrElse("")
```

Comment:
Should we ever even get here if any of these are None?  Wouldn't we be returning a nonsense URL if that was the case?